### DISCUSSION TITLE
`quarto install extension` from branch

### DISCUSSION THREAD

**Discussion by jmbuhr** [quarto-dev/quarto-cli#1914] [Feature Requests]
It would be handy for extension development and testing if you could use some syntax to specify a branch from which to download an extension via e.g. `quarto install extension <username>/<reponame>@<branch-or-tag>`

**Comment by jjallaire** [comment]
The tag version of this works now, the branch version should work soon (see https://github.com/quarto-dev/quarto-cli/issues/1836)

**Comment by mcanouil** [comment]
Closing as completed.
